pkg/worker/mount.go

```diff
@@ -21,7 +23,21 @@ func NewContainerMountManager() *ContainerMountManager {
 
 // SetupContainerMounts initializes any external storage for a container
 func (c *ContainerMountManager) SetupContainerMounts(request *types.ContainerRequest) error {
+	// Create local workspace path so we can symlink volumes before the container starts
+	os.MkdirAll(defaultContainerDirectory, os.FileMode(0755))
+
 	for i, m := range request.Mounts {
+		if m.MountPath == "/mnt/code" && request.Stub.Type.IsDeployment() {
+			source := m.LocalPath
+			localUserSource := tempUserCodeDir(request.ContainerId)
+			err := copyDirectory(source, localUserSource)
+			if err != nil {
+				log.Printf("<%s> - failed to eager copy juiceFS user code to local /mnt/code: %v\n", request.ContainerId, err)
```

Comment:
I wouldn't hardcode juicefs into the logs, its just whatever storage system we're currently using